### ISSUE TITLE
Nepal: Rasuwa Area

### ISSUE THREAD

**Issue by maning** [hotosm/imagery-requests#14]
Kathmandu Living Labs reported heavy damage in this area.
Approximate boundary of the area here: http://www.openstreetmap.org/relation/4583252#map=11/28.1646/85.3954&layers=H

**Comment by smit1678** [comment]
@maning We had covered some of this area in #7 with older pre-disaster imagery. 

Post-diaster seems spotty in this area from DG. imagery dated April 29 isn't fully covering that area and still some clouds: 

![](https://api.monosnap.com/rpc/file/download?id=X1sYCE6Nm1R2Xx50bc9iumoSAS0N8u)

![](https://api.monosnap.com/rpc/file/download?id=rLmpi8BeJtTdYAJvgrgDNLZbrAejkh)

**Comment by smit1678** [comment]
From skype chat, it seems that more north-east + Langtang national park are more priority areas. Or is the Betrawati area in the image above worth processing?

**Comment by pierzen** [comment]
yes langtang is high priority. Road blockages, avalanche, rock slidings, many deaths, villages gone. But there might be others south too.

**Comment by smit1678** [comment]
@pierzen Do you have a AOI that is reasonably sized to focus on?

**Comment by pierzen** [comment]
Would this help to see area to focus ?
these are emergencies reported
http://www.kathmandulivinglabs.org/earthquake/

**Comment by mikelmaron** [comment]
we really need a defined AOI for post event tasking....

**Comment by d3netxer** [comment]
can this issue be closed? I can't determine if this issue was covered by #7 or not
